### DISCUSSION TITLE
Missing support for FakeItEasy 7.0.0

### DISCUSSION THREAD

**Discussion by tmx7777** [AutoFixture/AutoFixture#1250] [Q&A] [answered]
After upgrading to FakeItEasy 7.0.0 I'm getting the following message: `Warning	NU1608	Detected package version outside of dependency constraint: AutoFixture.AutoFakeItEasy 4.15.0 requires FakeItEasy (>= 4.9.0 && < 7.0.0) but version FakeItEasy 7.0.0 was resolved.`

All of my unit tests are still running.

* Can I ignore those warnings for now?
* Will there be an update to extend the depedency constraint?

**Comment by aivascu** [comment, marked as answer]
@tmx7777 I'll update the package support for FakeItEasy 7 and we should have an updated package soon.
Hopefully there were no major breaking changes that affects the AutoFakeItEasy library.

**Update:**
Support for FakeItEasy `v7` was added in AutoFixture `v4.17.0`.

**Reply by tmx7777** [reply]
Thank you.

**Reply by aivascu** [reply]
@tmx7777 support for FakeItEasy `v7` was added in AutoFixture `v4.17.0`.
